app/models/sponsorship.rb

```diff
@@ -19,7 +19,7 @@ class Sponsorship < ApplicationRecord
   validates :status, presence: true, inclusion: { in: STATUSES }
   validates :url, url: { allow_blank: true, no_local: true, schemes: %w[http https] }
   validates :user, :organization, :featured_number, presence: true
-
+  validates :sponsorable_type, with: "validate_sponsorable_type"
```

Comment:
Rails has a different DSL method for this:

```suggestion
  validate :validate_sponsorable_type
```